### DISCUSSION TITLE
Overflow element

### DISCUSSION THREAD

**Discussion by rimzzlabs** [guilhermerodz/input-otp#36] [Q&A] [answered]
![image](https://github.com/guilhermerodz/input-otp/assets/62492410/60b367a9-8ba1-4fa3-81d9-08dc18f11e23)
What this element do? it add padding-end to the input container, and add horizontal scroll to my entire app

how do I hide that element?

**Comment by guilhermerodz** [comment]
hi @rimzzlabs that is definitely an issue. I am inspecting right now and will get back to you.
btw, this is supposed to be a password manager badge detector. just an invisible div.

- in which version are you in?
- which browser?
- have you done any CSS override or anything like that?

**Reply by rimzzlabs** [reply]
hi, i am running on
- input-otp @ ^1.2.0
- chromium browser (brave)
- already did that with `containerClassName="[&>div:first-child]:hidden"` with tailwind as temporary workaround

I knew this could be an issue but Im also afraid to put this to issue page

**Reply by guilhermerodz** [reply]
oh I finally got to reproduce the issue locally. @rimzzlabs thank you so much for reporting this.

I'm glad you found a temp workaround.

I really want to keep the password badge detector feature active by default, so I'm working on a fix ASAP to publish 1.2.1.

I believe a good internal fix is just placing that same weird div inside of another div with overflow:hidden. That way no one deals with extra padding.

Will let you know as soon as 1.2.1 is published

**Reply by rimzzlabs** [reply]
thank you @guilhermerodz , I'm glad I can help in any way I can

**Reply by guilhermerodz** [reply, marked as answer]
hi @rimzzlabs that is fixed on the new update v1.2.2, please run `npm i input-otp@latest`
